README.md

```diff
@@ -4,25 +4,78 @@
 
 # DevExpress Reporting for .NET MAUI - Generate Bound-Data Report
 
-...
+This example generates a data-bound report in the [DevExpress Report Designer](https://docs.devexpress.com/MAUI/404437/reporting/reporting-overview) and displays it in the [PDF Viewer for .NET MAUI](https://docs.devexpress.com/MAUI/404632/pdf-viewer/pdf-viewer). To use Reporting APIs in a .NET MAUI project, you need a license that includes [DevExpress Reporting](https://www.devexpress.com/subscriptions/reporting/).
 
 <img width="40%" alt="DevExpress Reporting for .NET MAUI - Generate Bound-Data Report" src="Images/app-preview.png">
 
 ## Included Controls and Their Properties
 
-...
+* [PdfViewer](https://docs.devexpress.com/MAUI/DevExpress.Maui.Pdf.PdfViewer): [ShowToolbar](https://docs.devexpress.com/MAUI/DevExpress.Maui.Pdf.PdfViewer.ShowToolbar), [DocumentSource](https://docs.devexpress.com/MAUI/DevExpress.Maui.Pdf.PdfViewer.DocumentSource)
 
 ## Implementation Details
 
-...
+1. Define the `CreateTableReportAsync()` function that creates a new report. Use the `XtraReportInstance()` constructor to generate a report instance. Call the [`CreateDocument()`](https://docs.devexpress.com/XtraReports/DevExpress.XtraReports.UI.XtraReport.CreateDocument) method to create a document from the report:
+
+    ```csharp
+    public async Task<XtraReport> CreateTableReportAsync() {
+        XtraReportInstance report = null;
+        await Task.Run(() =>
+        {
+            report = new XtraReportInstance() { Name = "Sample" };
+            report.CreateDocument();
+        });
+        return report;
+    }
+    ```
+
+2. Call the `CreateTableReportAsync()` function to generate a new report and get a path to the report:
+
+    ```csharp
+    private async void ContentPage_Loaded(object sender, EventArgs e) {
+        XtraReport report = await CreateTableReportAsync();
+        string resultFile = Path.Combine(FileSystem.Current.AppDataDirectory, report.Name + ".pdf");
```

Comment:
No need for this line - you have it in the next step (and it makes more sense there).

```suggestion

```